packages/cross_file/example/lib/readme_excerpts.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:cross_file/cross_file.dart';
```

Comment:
Ignores `avoid_print` lint for this file, since it's used to illustrate an example:

```suggestion

// ignore_for_file: avoid_print

import 'package:cross_file/cross_file.dart';
```